src/server/exemption/check-your-answers/controller.test.js

```diff
@@ -366,6 +367,71 @@ describe('check your answers controller', () => {
     expect(form.contains(submitButton)).toBe(true)
   })
 
+  test('should render the reason why it should be withheld from the public register', async () => {
+    const { result, statusCode } = await server.inject({
+      method: 'GET',
+      url: '/exemption/check-your-answers'
+    })
+
+    expect(statusCode).toBe(200)
+    const { document } = new JSDOM(result).window
+
+    const rows = document.querySelectorAll(
```

Comment:
Would be good to extract a couple of test helpers eg `getTextForRowNumber(2)` to reduce noise in the tests